websites/N/Netflix/presence.ts

```diff
@@ -77,21 +77,21 @@ presence.on("UpdateData", async () => {
     browsingStamp = Math.floor(Date.now() / 1000);
   }
 
-  if (!oldLang) {
+  if (!oldLang) 
     oldLang = newLang;
-  } else if (oldLang !== newLang) {
+   else if (oldLang !== newLang) {
     oldLang = newLang;
     strings = getStrings();
   }
 
   if (document.location.pathname.includes("/watch")) {
     const video: HTMLVideoElement = document.querySelector(
-      ".VideoContainer video"
+      ".watch-video--player-view video"
     );
```

Comment:
|| or case. I have old UI and `.watch-video--player-view video` is not a thing yet for me.

Reply:
Oops. missed this one. Thanks for checking.